test/functional/install/install-no-fullfile-fallbacks.bats

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bats
+
+load "../testlib"
+
+test_setup() {
+
+	create_test_environment "$TEST_NAME" 10
+	create_bundle -n test-bundle -f /file_1,/file_2 "$TEST_NAME"
+	sudo mkdir -p "$TARGETDIR"/usr/share/clear/bundles
+	sudo touch "$TARGETDIR"/usr/share/clear/bundles/test-bundle
+	sudo rm "$WEBDIR"/10/pack-test-bundle-from-0.tar
+	sudo rm "$WEBDIR"/10/pack-os-core-from-0.tar
+	sudo rm -rf "$WEBDIR"/10/files/*
+
+}
+
+@test "install bundle with no zero packs and no fullfile fallbacks" {
+
+	run sudo sh -c "$SWUPD verify $SWUPD_OPTS --install -m 10"
+
+	# no way to get content, everything should fail
+	assert_status_is 1
```

Comment:
At first I was confused why this status was compared against '1' instead of a defined error, but I see that there is no error defined for exit status 1 in swupd-error.h. Is there a reason that an entry for exit status 1 does not exist there?

Reply:
verify.c is returning EXIT_FAILURE(1) and it shouldn't. We have a ticket for that #610.
Not a problem in this patch though.